code/modules/antagonists/changeling/changeling.dm

```diff
@@ -396,7 +404,6 @@
 	to_chat(owner.current, "<b>You must complete the following tasks:</b>")
 	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ling_aler.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
 
-	owner.announce_objectives()
```

Comment:
why

Reply:
they show up on the popup anyways, but eh, i can add it back

Reply:
yea